crates/fuel-gas-price-algorithm/gas-price-analysis/src/optimisation.rs

```diff
@@ -1,14 +1,16 @@
 use super::*;
 
 fn da_pid_factors(size: usize) -> Vec<(i64, i64)> {
-    let mut rng = StdRng::seed_from_u64(10902);
-    (0usize..size)
-        .map(|_| {
-            let p = rng.gen_range(1_000..100_000_000_000);
-            let d = rng.gen_range(1_000..100_000_000_000);
-            (p, d)
-        })
-        .collect()
+    // let mut rng = StdRng::seed_from_u64(10902);
+    // (0usize..size)
+    //     .map(|_| {
+    //         let p = rng.gen_range(100_000..5_000_000);
+    //         let d = rng.gen_range(100_000..5_000_000);
+    //         (p, d)
+    //     })
+    //     .collect()
```

Comment:
Can we clean this up? Or is this helpful to keep around as a reference?

Reply:
We want to keep this. It's how we generate new values. I have followup issues that will use CLI commands to activate this section.